src/util/forge-config.js

```diff
@@ -14,7 +14,7 @@ export default async (dir) => {
   forgeConfig.make_targets = Object.assign({
     win32: ['squirrel'],
     darwin: ['zip'],
-    linux: ['deb', 'rpm'],
+    linux: ['deb', 'rpm', 'flatpak'],
```

Comment:
Same as before this would set `flatpak` to be a default for all users, is this what we want?